lua/smh/client/controller.lua

```diff
@@ -168,6 +168,18 @@ function CTRL.OpenHelp()
     gui.OpenURL("https://github.com/Winded/StopMotionHelper/blob/master/TUTORIAL.md")
 end
 
+function CTRL.IsRendering(rendering)
+	net.Start(SMH.MessageTypes.IsRendering)
```

Comment:
Mixing tabs and spaces, this causes identation problems with different editors, like in this Github view